src/Agent/NewRelic/Agent/Core/Api/AgentApi.cs

```diff
@@ -280,6 +281,22 @@ void work()
             TryInvoke(work, apiName, apiMetric);
         }
 
+        /// <summary>
+        /// Notice an error identified by a simple message and report it to the New Relic service.
+        /// If this method is called within a transaction,
+        /// the exception will be reported with the transaction when it finishes.  
```

Comment:
There is no C# `Exception` in this case, maybe change the verbiage to error, or clarify what this means? Are you referring to an `Exception` in the UI?